frameit/frames_generator/Rakefile

```diff
@@ -40,6 +57,10 @@ task(:generate_device_frames) do
     mv(output_file_path, sanitaized_file_path) unless output_file_path == sanitaized_file_path
     convert_image_resource(sanitaized_file_path)
     image_assets << sanitaized_basename
+
+    x, y, width = measure_slot(sanitaized_file_path)
+    device_name = sanitize_device_name(sanitaized_basename)
```

Comment:
typo on `sanitaized` from existing code

Reply:
@janpio  You mean we should fix `sanitized_file_path` and `sanitized__basename`? If yes, I will change that 🙂

Reply:
Definitely - no need to keep that misspelling around.